assembl/configs/base_env.yaml

```diff
@@ -178,3 +178,19 @@ maxretry: 10
 # Can be either a URL or a local filesystem path
 # IF URL, there should be no query strings
 theme_base: ""
+
+# The extensions that are allowed for uploaded files
+# on thread module
+# To be completed if needed
+allowed_extensions:
```

Comment:
What about `csv`, `tsv`, `txt`, `avi`, `mpeg`, `mpg`, `mp4`, `mp3`, `wav`, etc etc. For images, what about `jpe`, `svg`, `bmp`, or google's new `webp`? What if people want to share zip files (which inside could have malicious things inside)? This list is far too limiting as a default. Think about the full mime-type list.

If this list becomes unbearably long, inverting the logic may be a better option. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types
https://www.sitepoint.com/mime-types-complete-list/ 😱

Reply:
Yes, I agree that this list may be too limited, we should discuss what we want to include. We must stick with the whitelist approach, because blacklist is not a security best practice: what if you block `.php` but the attacker send `.php5` file?
The list of mime-types is really huge, but I don't think we will need more than 10-15 extensions at most. It's in the config, so we can change it easily.

Reply:
Maybe you should whitelist allowed mimetypes, and not extensions, this way you can allow `image/*` or `video/*` if you want.

Reply:
Done with possibility to use `image/*` and also specific like `application/pdf`.